src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -13,68 +13,8 @@
 
 public class FilterTextFieldTest extends UITest {
 
-    private void testCompletions(FilterTextField field) {
-        // Basic completion
-        clearField();
-        type("cou").push(KeyCode.TAB);
-        awaitCondition(() -> field.getText().equals("count"));
-
-        // Completion does not only work for alternating keys typed
-        clearField();
-        type("c");
-        awaitCondition(() -> field.getSelectedText().equals("losed"));
-        type("l");
-        awaitCondition(() -> field.getSelectedText().equals("osed"));
-        type("o");
-        awaitCondition(() -> field.getSelectedText().equals("sed"));
-        type(KeyCode.TAB);
-        awaitCondition(() -> field.getSelectedText().equals(""));
-        awaitCondition(() -> field.getText().equals("closed"));
-
-        // Completion with selection
-        clearField();
-        type("cou").push(KeyCode.TAB);
-        push(KeyCode.LEFT);
-        for (int i = 0; i < 3; i++) {
-            field.selectBackward();
-        }
-        // c[oun]t
-        type("lo").push(KeyCode.TAB); // 'c' + 'lo' is a prefix of 'closed'
-        awaitCondition(() -> field.getText().equals("closedt"));
-    }
-
-    private void testSpaces(FilterTextField field) {
-        // Consecutive spaces allowed
-        clearField();
-        type("cou").push(KeyCode.TAB);
-        type("   ");
-        awaitCondition(() -> field.getText().equals("count   "));
-
-        // Insertion of spaces before spaces
-        clearField();
-        type("assi").push(KeyCode.TAB);
-        type(" c").push(KeyCode.BACK_SPACE); // cancel completion
-        push(KeyCode.LEFT, 2);
-        type(" ");
-        awaitCondition(() -> field.getText().equals("assignee  c"));
-
-        // Insertion of spaces after spaces
-        clearField();
-        type("assi").push(KeyCode.TAB);
-        type(" c").push(KeyCode.BACK_SPACE); // cancel completion
-        push(KeyCode.LEFT);
-        type(" ");
-        awaitCondition(() -> field.getText().equals("assignee  c"));
-
-        // Insertion of spaces with trailing spaces
-        clearField();
-        type("assignee ");
-        push(KeyCode.LEFT, 2);
-        type(" ");
-        push(KeyCode.LEFT, 8);
-        type(" ");
-        awaitCondition(() -> field.getText().equals(" assigne e "));
-    }
+    private static final String VALID_FILTER_STYLE = "-fx-control-inner-background: white";
+    private static final String INVALID_FILTER_STYLE = "-fx-control-inner-background: #EE8993";
```

Comment:
Rather than redefine these, maybe we could use the existing constants from FilterTextField?